app/src/main/java/com/github/h3lp3rs/h3lp/model/forum/implementation/SimpleDBForum.kt

```diff
@@ -61,14 +63,14 @@ abstract class SimpleDBForum(private val rootForum : Database) : Forum {
      * @param currentTime The current date-time
      * @return Formatted current date-time
      */
-    @RequiresApi(Build.VERSION_CODES.O)
-    private fun getFormattedPostTime(currentTime : ZonedDateTime) : String {
+
+    private fun getFormattedPostTime(currentTime: ZonedDateTime): String {
```

Comment:
BUG REPORT:
When the user sets its time on:
14:00 instead of 2 PM (format), the app does no longer work. Indeed, we suppose AM/PM style for our app. This is a wrong choice.

Reply:
Did you have a framework to test this? I'd like to find a solution but don't know how to actually check if it works

Reply:
No it was only checked irl.
Can't we just use a predefined more robust formatter directly from the Date class?